### DISCUSSION TITLE
Is there any way to remove or hide action icons from a tab?

### DISCUSSION THREAD

**Discussion by Dozorengel** [gitkraken/vscode-gitlens#2011] [Q&A] [answered]
As far as I understand, these action icons appear with GitLens. Is there any way to remove or hide them from a tab? They take up a lot of space. Moreover, with 'Wrap tabs' option enabled it makes tabs jump up a lot each time I switch over the tab groups.
![image](https://user-images.githubusercontent.com/40327659/170711137-ff4b84ed-2803-43f4-84ba-505db3e6f21a.png)

**Comment by eamodio** [comment, marked as answer]
Yes, open the GitLens interactive settings editor from the Command Palette (<kbd>F1</kbd> or <kbd>ctrl/cmd</kbd>+<kbd>shift</kbd>+<kbd>p</kbd>) and search for _GitLens: Open Settings_. From there jump down to the _Menus & Toolbars_ section and you can toggle on and off any of the GitLens menus/toolbars

![image](https://user-images.githubusercontent.com/641685/171325122-443128ce-bf1c-45c5-a0ae-5c1a96271d0f.png)
